pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -0,0 +1,68 @@
+"""
+Function to download the IGPP Global Earth Free-Air Anomaly datasets from the
+GMT data server, and load as :class:`xarray.DataArray`.
+
+The grids are available in various resolutions.
+"""
+from pygmt.datasets.load_remote_dataset import _load_remote_dataset
+from pygmt.helpers import kwargs_to_strings
+
+
+@kwargs_to_strings(region="sequence")
+def load_earth_free_air_anomaly(resolution="01d", region=None, registration=None):
+    r"""
+    Load an Earth Free-Air Anomaly grid in various resolutions.
+
+    The grids are downloaded to a user data directory
+    (usually ``~/.gmt/server/earth/earth_faa/``) the first time you invoke
+    this function. Afterwards, it will load the grid from the data directory.
+    So you'll need an internet connection the first time around.
+
+    These grids can also be accessed by passing in the file name
+    **@earth_faa**\_\ *res*\[_\ *reg*] to any grid plotting/processing
+    function. *res* is the grid resolution (see below), and *reg* is grid
+    registration type (**p** for pixel registration or **g** for gridline
+    registration).
+
+    Refer to :gmt-datasets:`earth-faa.html` for more details.
+
+    Parameters
+    ----------
+    resolution : str
+        The grid resolution. The suffix ``d`` and ``m`` stand for
+        arc-degree and arc-minute. It can be ``"01d"``, ``"30m"``,
+        ``"20m"``, ``"15m"``, ``"10m"``, ``"06m"``, ``"05m"``, ``"04m"``,
+        ``"03m"``, ``"02m"``, or ``"01m"``.
+
+    region : str or list
+        The subregion of the grid to load, in the forms of a list
+        [*xmin*, *xmax*, *ymin*, *ymax*] or a string *xmin/xmax/ymin/ymax*.
+        Required for grids with resolutions higher than 5
+        arc-minute (i.e., ``"05m"``).
```

Comment:
```suggestion
        arc-minutes (i.e., ``"05m"``).
```

Reply:
Looks like there is some inconsistency regarding using singular or plural and this has to be changed also for the other datasets, e.g. [load_earth_age](https://www.pygmt.org/dev/api/generated/pygmt.datasets.load_earth_age.html?highlight=arc-minute), [load_earth_geoid](https://www.pygmt.org/dev/api/generated/pygmt.datasets.load_earth_geoid.html?highlight=arc-minute), etc. I can submit a separate PR, if we agree that plural is better as suggested by @michaelgrund.